packages/core/src/lib/Cache.ts

```diff
@@ -166,31 +129,19 @@ export class CachingCategory<Event, State, Context> implements ICategory<Event,
   ) {}
 
   private cacheKey(streamId: string) {
-    return `${this.categoryName}/${streamId}`
+    return this.strategy.cacheKey(`${this.categoryName}/${streamId}`)
```

Comment:
if you can borrow a pre-concatenated form from the read path, it will reduce garbage